app/models/member.rb

```diff
@@ -157,6 +157,11 @@ def party_name
     party_object.long_name
   end
 
+  # Long version of party name ukr_version
+  def display_party_name_ukr
```

Comment:
My pleasure :smiley: 

This is ready for merge. Do you want to merge and deploy it this time?